app/apps/handlers.js

```diff
@@ -50,10 +49,14 @@ exports.list = (req, res, next) => {
 exports.details = (req, res, next) => {
   Promise.all([App.get(req.params.id), Bucket.list(), User.list()])
     .then(([app, buckets, users]) => {
+      const user_ids = app.data.userapps.map(userapp => userapp.user.auth0_id);
+      const current_user_is_app_admin = user_ids.includes(req.user.user_id);
```

Comment:
This code might be better placed in the User model, eg:
```javascript
const current_user_is_app_admin = app.has_admin(req.user.user_id)
```

Reply:
👍

Reply:
❓